Cargo.toml

```diff
@@ -19,6 +19,7 @@ hound = "3.4.0"
 log = "0.4.14"
 realfft = "1.1.0"
 crossbeam-channel = "0.5.1"
+dasp = {version = "0.11", features=["ring_buffer"]}
```

Comment:
the moment we start pulling in dasp, we might as build this whole library on top of dasp_graph, dash_frame, dasp_sample etc ;)